production.Dockerfile

```diff
@@ -79,27 +79,35 @@ RUN apk --update --no-cache --virtual .build-deps add \
 # - we need few additional OS packages for this. Let's install
 #   and then uninstall them when the compilation is completed.
 COPY package.json yarn.lock ./
-COPY plugins/package.json plugins/yarn.lock ./plugins/
+COPY ./plugin-server/ ./plugin-server/
```

Comment:
I had some conflicts with you @guidoiaquinti so you might want to quickly skim this over

Reply:
the file I mean